build.sbt

```diff
@@ -8,16 +8,17 @@ version := "1.0-SNAPSHOT"
 
 scalaVersion := "2.10.6"
 
-lazy val jvmSdkVersion = "1.0.0-RC1"
+lazy val jvmSdkVersion = "1.0.0"
+lazy val jvmSdkScalaVersion = "1.0.0-RC3"
 
 libraryDependencies ++= Seq(
   javaCore,
   javaJdbc,
   javaWs,
-  "com.commercetools.sdk.jvm.core" % "commercetools-models" % jvmSdkVersion withSources(),
-  "com.commercetools.sdk.jvm.core" % "commercetools-java-client" % jvmSdkVersion withSources(),
-  "com.commercetools.sdk.jvm.core" % "commercetools-convenience" % jvmSdkVersion withSources(),
-  "com.commercetools.sdk.jvm.scala-add-ons" %% "commercetools-play-2_4-java-client" % jvmSdkVersion withSources(),
+  "com.commercetools.sdk.jvm.core" % "commercetools-models" % jvmSdkVersion,
```

Comment:
We removed it because it didn't compile. Now after I had a second look at it was just one line that made problems
`"com.commercetools.sdk.jvm.core" % "commercetools-java-client" % jvmSdkVersion withSources(),`
I think that line is not even needed. And we could re-add the other `withSources`